packages/models/src/lib/implementation/schemas.ts

```diff
@@ -124,7 +125,7 @@ export function positiveIntSchema(description: string) {
  * @param description
  */
 export function unixFilePathSchema(description: string) {
-  return z.string({ description }).regex(unixFilePathRegex);
+  return z.string({ description }).min(1);
```

Comment:
I don't it makes sense to have a separate `generalFilePathSchema` and `unixFilePathSchema` at this point, they're now exactly the same except for the `.trim()` and custom error message. I suggest we just have one `filePathSchema`.

Reply:
done